### PR TITLE
Make strorageType required for COS import

### PR DESCRIPTION
**Pull request by michaelkad**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
=== RUN   TestAccIBMPIImageCOSPublicImport

--- PASS: TestAccIBMPIImageCOSPublicImport (1185.82s)
PASS
```

https://jsw.ibm.com/browse/POWERIAAS-9917

### CODE REVIEW

File: ibm/service/power/resource_ibm_pi_image.go [hunk 1]
```diff
@@ -111,10 +111,11 @@ func ResourceIBMPIImage() *schema.Resource {
 				ForceNew:      true,
 			},
 			helpers.PIImageStorageType: {
```
Comment (ismirlia) [inline ibm/service/power/resource_ibm_pi_image.go:113]: Are we not relabeling helpers.PIImageStorageType to helpers.PIImageStorageTier to match the new CLI wording?

Comment (michaelkad) [reply ibm/service/power/resource_ibm_pi_image.go:113]: We discussed this — We will have to open a jira for all those changes.

**Review by ismirlia — APPROVED** [review]
Looks good!

File: website/docs/r/pi_image.html.markdown [hunk 2]
```diff
@@ -79,7 +79,7 @@ Review the argument references that you can specify for your resource.
 - `pi_image_secret_key` - (Optional, String, Sensitive) Cloud Object Storage secret key; required for buckets with private access.
   - `pi_image_secret_key` is required with `pi_image_access_key`
 - `pi_image_storage_pool` - (Optional, String) Storage pool where the image will be loaded, if provided then `pi_image_storage_type` and `pi_affinity_policy` will be ignored.
-- `pi_image_storage_type` - (Optional, String) Type of storage. Will be ignored if `pi_image_storage_pool` or `pi_affinity_policy` is provided. If only using `pi_image_storage_type` for storage selection then the storage pool with the most available space will be selected.
+- `pi_image_storage_type` - (Optional, String) Type of storage. Required if `pi_image_storage_pool` or `pi_affinity_policy` is provided when importing an image from cloud storage.
```
Comment (yussufsh) [inline website/docs/r/pi_image.html.markdown:82]: I do not understand this change.
As per the description in swagger referred from https://github.com/IBM-Cloud/power-go-client/blob/master/power/models/create_cos_image_import_job.go#L59 this is not changing.

**Review by yussufsh — CHANGES_REQUESTED** [review]

File: website/docs/r/pi_image.html.markdown (hunk 2 above)
Comment (michaelkad) [reply website/docs/r/pi_image.html.markdown:82]: @yussufsh This is what the message would be after we GA. It is still the same in SB, I will keep this as it is and would update if it is different than the final message.

**Comment by michaelkad** [comment]
Added in #10

**Pull request closed without merging**